catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveCommandTest.java

```diff
@@ -91,6 +101,39 @@ protected SolrCacheMBean getCacheProxy() {
         verify(mbean, times(1)).removeById(idsArray);
     }
 
+    @Test
+    public void testOverMaxItemList() throws Exception {
+        ConsoleOutput consoleOutput = mock(ConsoleOutput.class);
+        int totalResultsToDelete = 100;
+        int resultsPerCFQuery = 22;
+        int numCFCalls = 5;
+        List<String> ids = new ArrayList<>();
+        List<Metacard> mcList = getMetacardList(totalResultsToDelete);
```

Comment:
Also might consider making the test vars into declared class fields if they are reused across more future tests. Then you can instantiate them in a Setup method and remove some redundancy. 

Agree with @josephthweatt on removing the abbreviations from the variable names to help with readability.